ui/imports/utils/Utils.qml

```diff
@@ -700,6 +700,28 @@ QtObject {
         xhr.send();
     }
 
+    function getProfileType(isMe, isBridgedAccount, isBlocked) {
+        if (isMe)
+            return Constants.profileType.self
+        if (isBridgedAccount)
+            return Constants.profileType.bridged
+
+        return isBlocked ? Constants.profileType.blocked
+                         : Constants.profileType.regular
+    }
+
+    function getContactType(contactRequestState, isContact) {
```

Comment:
This is weird (and maybe duplicate); we can infer the `isContact` already if `contactRequestState === Constants.ContactRequestState.Mutual`